iree/turbine/kernel/ops/wave_ops.py

```diff
@@ -1153,6 +1166,33 @@ class SchedulingBarrier(CustomOp):
     operations: list[Operation]
 
 
+@define_op("atomic_min")
+@dataclass
+class AtomicMin(BinaryOpBase, ABC):
+    """
+    Represents an atomic operation in the graph.
+    Takes in value and buffer to perform the operation on.
```

Comment:
Can you add a bit more detail WRT how buffer is a memory type (e.g global/shared) and value is a register